src/java/Transport.java

```diff
@@ -0,0 +1,30 @@
+package java;
+
+public abstract class Transport {
+    int power;
+    int maksSpeed;
+    int weight;
+    String carModel;
+
+    public Transport(int power, int maksSpeed, int weight, String carModel) {
+        this.power = power;
+        this.maksSpeed=maksSpeed;
+        this.weight=weight;
+        this.carModel=carModel;
```

Comment:
@Hlieb Старайся отделять пробелами операторы
```suggestion
        this.maksSpeed = maksSpeed;
        this.weight = weight;
        this.carModel = carModel;
```